### ISSUE TITLE
macOS keychain alert shown on first launch

### ISSUE THREAD

**Issue by frankdilo** [microsoft/appcenter-sdk-apple#1655]
### **Description**

When launching an app on macOS that uses the AppCenter SDK the user is prompted to allow keychain access (see screenshot below). This is uncommon for mac apps and user-hostile, and it won't allow me to ship the AppCenter SDK within my apps anymore unless fixed.

![image](https://user-images.githubusercontent.com/715405/59187540-f6d37b00-8b75-11e9-8b42-00d29d7db42a.png)

### **Repro Steps**

Please list the steps used to reproduce your issue.

1. Create a new macOS app
2. Integrate AppCenter
3. Launch the app and see the alert appear

### **Details**

1. Which SDK version are you using?
    - 2.0.1
2. Which OS version did you experience the issue on?
    - macOS 10.14
3. Which Cocoapods version are you using (run `pod --version`)?
    - 1.7
4. What device version did you see this error on?  Were you using an emulator or a physical device?
    - macOS laptop
5. What language are you using?
    - [ ] Objective C
    - [x] Swift
6. What third party libraries are you using?
7. Please enable verbose logging for your app using `MSAppCenter.setLogLevel(.verbose)` before your call to `MSAppCenter.start(...)` for Swift, or `[MSAppCenter setLogLevel:MSLogLevelVerbose]` before `[MSAppCenter start: ...]` for Objective C and include the logs here:

```2019-06-10 11:53:34.514226+0200 Boxy for Contacts[21961:1556149] [AppCenter] VERBOSE: -[MSDelegateForwarder addTraceBlock:]_block_invoke_2/88 Start buffering traces.
2019-06-10 11:53:34.514269+0200 Boxy for Contacts[21961:1556149] [AppCenter] DEBUG: -[MSDelegateForwarder setEnabledFromPlistForKey:]_block_invoke/278 Delegate forwarder for info.plist key 'AppCenterAppDelegateForwarderEnabled' enabled. It may use swizzling.
2019-06-10 11:53:34.514288+0200 Boxy for Contacts[21961:1556149] [AppCenter] VERBOSE: +[MSDelegateForwarder flushTraceBuffer]/106 Stop buffering traces, flushed.
2019-06-10 11:53:34.514711+0200 Boxy for Contacts[21961:1556149] [AppCenter] DEBUG: -[MSSessionContext init]/42 1 session(s) in the history.
2019-06-10 11:53:34.516196+0200 Boxy for Contacts[21961:1556149] [AppCenter] VERBOSE: -[MSSessionContext setSessionId:]/66 Stored new session with id:(null) and timestamp: 2019-06-10 09:53:34 +0000.
2019-06-10 11:53:34.516648+0200 Boxy for Contacts[21961:1556149] [AppCenter] DEBUG: -[MSUserIdContext init]/54 1 userId(s) in the history.
2019-06-10 11:53:34.517985+0200 Boxy for Contacts[21961:1556149] [AppCenter] INFO: -[MSHttpIngestion networkStateChanged]/356 Internet connection is up.
2019-06-10 11:53:34.518234+0200 Boxy for Contacts[21961:1556149] [AppCenter] INFO: -[MSHttpIngestion networkStateChanged]/356 Internet connection is up.
2019-06-10 11:53:34.518259+0200 Boxy for Contacts[21961:1556149] [AppCenter] INFO: -[MSHttpIngestion networkStateChanged]/356 Internet connection is up.
2019-06-10 11:53:34.558786+0200 Boxy for Contacts[21961:1556149] [AppCenter] INFO: -[MSAppCenter configureWithAppSecret:transmissionTargetToken:fromApplication:]/281 App Center SDK configured successfully.
2019-06-10 11:53:34.582305+0200 Boxy for Contacts[21961:1556371] [AppCenter] DEBUG: -[MSAuthTokenContext authTokenHistory]/228 Retrieved history state.
2019-06-10 11:53:34.583502+0200 Boxy for Contacts[21961:1556371] [AppCenter] VERBOSE: -[MSLogDBStorage loadLogsWithGroupId:limit:excludedTargetKeys:afterDate:beforeDate:completionHandler:]/212 Load log(s) with id(s) '166' as batch Id:5ED71F0D-7645-49D8-930F-586938A12E79
2019-06-10 11:53:34.584259+0200 Boxy for Contacts[21961:1556371] [AppCenter] DEBUG: -[MSChannelUnitDefault sendLogContainer:withAuthTokenFromArray:atIndex:]/219 Sending 1/1 log, group Id: AppCenter, batch Id: 5ED71F0D-7645-49D8-930F-586938A12E79, session Id: (null), payload:
{
  "device" : {
    "screenSize" : "1200x1920",
    "appBuild" : "3",
    "osName" : "macOS",
    "timeZoneOffset" : 120,
    "osVersion" : "10.14.4",
    "locale" : "en_US",
    "appNamespace" : "co.boxyapp.contacts",
    "osBuild" : "18E226",
    "sdkName" : "appcenter.macos",
    "oemName" : "Apple",
    "model" : "MacBookPro14,3",
    "sdkVersion" : "2.0.1",
    "appVersion" : "1.0"
  },
  "timestamp" : "2019-06-10T09:53:11.848Z",
  "type" : "startService",
  "services" : [
    "Crashes",
    "Analytics"
  ]
}
2019-06-10 11:53:34.584657+0200 Boxy for Contacts[21961:1556371] [AppCenter] VERBOSE: -[MSAppCenterIngestion createRequest:eTag:authToken:]/91 URL: https://in.appcenter.ms/logs?api-version=1.0.0
2019-06-10 11:53:34.584750+0200 Boxy for Contacts[21961:1556371] [AppCenter] VERBOSE: -[MSAppCenterIngestion createRequest:eTag:authToken:]/92 Headers: Install-ID = 92F24F44-11CE-47C7-9F26-EA81F33AA2DC, App-Secret = ****************************6f7334e2, Content-Type = application/json
2019-06-10 11:53:34.588511+0200 Boxy for Contacts[21961:1556149] [AppCenter] VERBOSE: -[MSAppCenter start:withServices:fromApplication:]/296 Start services MSCrashes, MSAnalytics from an application
2019-06-10 11:53:34.588807+0200 Boxy for Contacts[21961:1556149] [AppCenterCrashes] DEBUG: -[MSCrashes configureCrashReporterWithUncaughtExceptionHandlerEnabled:]/562 EnableUncaughtExceptionHandler is set to YES
2019-06-10 11:53:34.588841+0200 Boxy for Contacts[21961:1556149] [AppCenterCrashes] VERBOSE: -[MSCrashes configureCrashReporterWithUncaughtExceptionHandlerEnabled:]/572 Enabled Mach exception handler.
2019-06-10 11:53:34.596238+0200 Boxy for Contacts[21961:1556149] [AppCenterCrashes] WARNING: -[MSCrashes configureCrashReporterWithUncaughtExceptionHandlerEnabled:]/586 Detecting crashes is NOT enabled due to running the app with a debugger attached.
2019-06-10 11:53:34.597375+0200 Boxy for Contacts[21961:1556149] [AppCenter] VERBOSE: -[MSSessionContext clearSessionHistoryAndKeepCurrentSession:]/88 Cleared old sessions.
2019-06-10 11:53:34.597805+0200 Boxy for Contacts[21961:1556149] [AppCenter] VERBOSE: -[MSUserIdContext clearUserIdHistory]/124 Cleared old userIds while keeping current userId.
2019-06-10 11:53:34.597832+0200 Boxy for Contacts[21961:1556149] [AppCenterCrashes] INFO: -[MSCrashes applyEnabledState:]/330 Crashes service has been enabled but the service cannot detect crashes due to running the application with a debugger attached.
2019-06-10 11:53:34.629836+0200 Boxy for Contacts[21961:1556149] [AppCenterCrashes] VERBOSE: -[MSCrashes startWithChannelGroup:appSecret:transmissionTargetToken:fromApplication:]/371 Started crash service.
2019-06-10 11:53:34.636293+0200 Boxy for Contacts[21961:1556149] [AppCenter] VERBOSE: -[MSSessionContext setSessionId:]/66 Stored new session with id:B335CCFA-49BD-4B34-862F-16296B62ED27 and timestamp: 2019-06-10 09:53:34 +0000.
2019-06-10 11:53:34.636353+0200 Boxy for Contacts[21961:1556149] [AppCenterAnalytics] INFO: -[MSSessionTracker renewSessionId]/49 New session ID: B335CCFA-49BD-4B34-862F-16296B62ED27
2019-06-10 11:53:34.636930+0200 Boxy for Contacts[21961:1556352] [AppCenter] VERBOSE: -[MSLogDBStorage loadLogsWithGroupId:limit:excludedTargetKeys:afterDate:beforeDate:completionHandler:]/212 Load log(s) with id(s) '165' as batch Id:35F9CA2C-01F5-4DF7-A6AC-AC561D8DC04E
2019-06-10 11:53:34.637126+0200 Boxy for Contacts[21961:1556352] [AppCenter] DEBUG: -[MSChannelUnitDefault sendLogContainer:withAuthTokenFromArray:atIndex:]/219 Sending 1/1 log, group Id: Analytics, batch Id: 35F9CA2C-01F5-4DF7-A6AC-AC561D8DC04E, session Id: 5EED0677-268A-442D-B3FD-F2961B700E00, payload:
{
  "sid" : "5EED0677-268A-442D-B3FD-F2961B700E00",
  "timestamp" : "2019-06-10T09:49:17.812Z",
  "device" : {
    "screenSize" : "1200x1920",
    "appBuild" : "3",
    "osName" : "macOS",
    "timeZoneOffset" : 120,
    "osVersion" : "10.14.4",
    "locale" : "en_US",
    "appNamespace" : "co.boxyapp.contacts",
    "osBuild" : "18E226",
    "sdkName" : "appcenter.macos",
    "oemName" : "Apple",
    "model" : "MacBookPro14,3",
    "sdkVersion" : "2.0.1",
    "appVersion" : "1.0"
  },
  "type" : "startSession"
}
2019-06-10 11:53:34.637349+0200 Boxy for Contacts[21961:1556352] [AppCenter] VERBOSE: -[MSAppCenterIngestion createRequest:eTag:authToken:]/91 URL: https://in.appcenter.ms/logs?api-version=1.0.0
2019-06-10 11:53:34.637449+0200 Boxy for Contacts[21961:1556352] [AppCenter] VERBOSE: -[MSAppCenterIngestion createRequest:eTag:authToken:]/92 Headers: Install-ID = 92F24F44-11CE-47C7-9F26-EA81F33AA2DC, App-Secret = ****************************6f7334e2, Content-Type = application/json
2019-06-10 11:53:34.638628+0200 Boxy for Contacts[21961:1556352] [AppCenter] VERBOSE: -[MSChannelUnitDefault resumeWithIdentifyingObjectSync:]/493 Identifying object <MSAnalytics: 0x6000029253b0> removed from pause lane for channel Analytics.
2019-06-10 11:53:34.638676+0200 Boxy for Contacts[21961:1556352] [AppCenter] DEBUG: -[MSChannelUnitDefault resumeWithIdentifyingObjectSync:]/495 Resume channel Analytics.
2019-06-10 11:53:34.639729+0200 Boxy for Contacts[21961:1556149] [AppCenterCrashes] VERBOSE: -[MSCrashes channel:didPrepareLog:internalId:flags:]/441 Storing a log to Crashes Buffer: (sid: B335CCFA-49BD-4B34-862F-16296B62ED27, type: startSession)
2019-06-10 11:53:34.639780+0200 Boxy for Contacts[21961:1556149] [AppCenterCrashes] VERBOSE: -[MSCrashes channel:didPrepareLog:internalId:flags:]/452 Found an empty buffer position.
2019-06-10 11:53:34.639884+0200 Boxy for Contacts[21961:1556149] [AppCenterAnalytics] INFO: -[MSAnalytics applyEnabledState:]/146 Analytics service has been enabled.
2019-06-10 11:53:34.641611+0200 Boxy for Contacts[21961:1556149] [AppCenterAnalytics] VERBOSE: -[MSAnalytics startWithChannelGroup:appSecret:transmissionTargetToken:fromApplication:]/99 Started Analytics service.
2019-06-10 11:53:34.641826+0200 Boxy for Contacts[21961:1556352] [AppCenter] DEBUG: -[MSChannelUnitDefault enqueueItem:flags:]_block_invoke/182 Saving log, type: startSession, flags: 1.
2019-06-10 11:53:34.642691+0200 Boxy for Contacts[21961:1556149] [AppCenterCrashes] VERBOSE: -[MSCrashes channel:didPrepareLog:internalId:flags:]/441 Storing a log to Crashes Buffer: (sid: (null), type: startService)
2019-06-10 11:53:34.642744+0200 Boxy for Contacts[21961:1556149] [AppCenterCrashes] VERBOSE: -[MSCrashes channel:didPrepareLog:internalId:flags:]/464 Remembering index 0 for oldest timestamp 1560160414.639769.
2019-06-10 11:53:34.642784+0200 Boxy for Contacts[21961:1556149] [AppCenterCrashes] VERBOSE: -[MSCrashes channel:didPrepareLog:internalId:flags:]/452 Found an empty buffer position.
2019-06-10 11:53:34.644983+0200 Boxy for Contacts[21961:1556352] [AppCenter] VERBOSE: -[MSLogDBStorage saveLog:withGroupId:flags:]_block_invoke/120 Log is stored with id: '167'
2019-06-10 11:53:34.645190+0200 Boxy for Contacts[21961:1556352] [AppCenterCrashes] VERBOSE: -[MSCrashes channel:didCompleteEnqueueingLog:internalId:]/494 Deleting a log from buffer with id 7802F9F5-1CEF-4AB4-B607-99462148DC63
2019-06-10 11:53:34.645356+0200 Boxy for Contacts[21961:1556352] [AppCenter] VERBOSE: -[MSChannelUnitDefault resumeWithIdentifyingObjectSync:]/493 Identifying object <MSAnalytics: 0x6000029253b0> removed from pause lane for channel Analytics/one.
2019-06-10 11:53:34.645391+0200 Boxy for Contacts[21961:1556352] [AppCenter] DEBUG: -[MSChannelUnitDefault resumeWithIdentifyingObjectSync:]/495 Resume channel Analytics/one.
2019-06-10 11:53:34.646687+0200 Boxy for Contacts[21961:1556352] [AppCenter] DEBUG: -[MSChannelUnitDefault enqueueItem:flags:]_block_invoke/182 Saving log, type: startService, flags: 1.
2019-06-10 11:53:34.649196+0200 Boxy for Contacts[21961:1556352] [AppCenter] VERBOSE: -[MSLogDBStorage saveLog:withGroupId:flags:]_block_invoke/120 Log is stored with id: '168'
2019-06-10 11:53:34.649408+0200 Boxy for Contacts[21961:1556352] [AppCenterCrashes] VERBOSE: -[MSCrashes channel:didCompleteEnqueueingLog:internalId:]/494 Deleting a log from buffer with id F50E5782-798B-4EDE-B812-FDF4C520A7FF
2019-06-10 11:53:35.173157+0200 Boxy for Contacts[21961:1556371] [AppCenter] VERBOSE: -[MSHttpIngestion sendCallAsync:]_block_invoke/256 HTTP response received with status code: 200, payload:
CorrelationId: e8c0dde6-c591-4954-98d2-411e6fd2fbb0  ReasonCode: Success
2019-06-10 11:53:35.173327+0200 Boxy for Contacts[21961:1556371] [AppCenter] INFO: -[MSHttpIngestion call:completedWithResult:]/305 Removed call id:5ED71F0D-7645-49D8-930F-586938A12E79 from pending calls:{
    "35F9CA2C-01F5-4DF7-A6AC-AC561D8DC04E" = "<MSIngestionCall: 0x600002628840>";
}
2019-06-10 11:53:35.173331+0200 Boxy for Contacts[21961:1556377] [AppCenter] DEBUG: -[MSChannelUnitDefault sendLogContainer:withAuthTokenFromArray:atIndex:]_block_invoke/242 Log(s) sent with success, batch Id:5ED71F0D-7645-49D8-930F-586938A12E79.
2019-06-10 11:53:35.180926+0200 Boxy for Contacts[21961:1556377] [AppCenter] VERBOSE: +[MSLogDBStorage deleteLogsFromDBWithColumnValues:columnName:inOpenedDatabase:]/353 Deletion of log(s) by id with value(s) '166' succeeded.
2019-06-10 11:53:36.019202+0200 Boxy for Contacts[21961:1556352] [AppCenter] VERBOSE: -[MSHttpIngestion sendCallAsync:]_block_invoke/256 HTTP response received with status code: 200, payload:
CorrelationId: 82418a25-1b61-46d5-9044-1663e4a8e99e  ReasonCode: Success
2019-06-10 11:53:36.019267+0200 Boxy for Contacts[21961:1556352] [AppCenter] INFO: -[MSHttpIngestion call:completedWithResult:]/305 Removed call id:35F9CA2C-01F5-4DF7-A6AC-AC561D8DC04E from pending calls:{
}
2019-06-10 11:53:36.019279+0200 Boxy for Contacts[21961:1556377] [AppCenter] DEBUG: -[MSChannelUnitDefault sendLogContainer:withAuthTokenFromArray:atIndex:]_block_invoke/242 Log(s) sent with success, batch Id:35F9CA2C-01F5-4DF7-A6AC-AC561D8DC04E.
2019-06-10 11:53:36.026759+0200 Boxy for Contacts[21961:1556377] [AppCenter] VERBOSE: +[MSLogDBStorage deleteLogsFromDBWithColumnValues:columnName:inOpenedDatabase:]/353 Deletion of log(s) by id with value(s) '165' succeeded.
decidePolicyFor https://contacts.google.com/u/2/
didStartProvisionalNavigation
didCommitNavigation
2019-06-10 11:53:37.647623+0200 Boxy for Contacts[21961:1556377] [AppCenter] VERBOSE: -[MSLogDBStorage loadLogsWithGroupId:limit:excludedTargetKeys:afterDate:beforeDate:completionHandler:]/212 Load log(s) with id(s) '167' as batch Id:19157BE4-E70C-46A2-9EB2-166F9C1244F0
2019-06-10 11:53:37.648053+0200 Boxy for Contacts[21961:1556377] [AppCenter] DEBUG: -[MSChannelUnitDefault sendLogContainer:withAuthTokenFromArray:atIndex:]/219 Sending 1/1 log, group Id: Analytics, batch Id: 19157BE4-E70C-46A2-9EB2-166F9C1244F0, session Id: B335CCFA-49BD-4B34-862F-16296B62ED27, payload:
{
  "sid" : "B335CCFA-49BD-4B34-862F-16296B62ED27",
  "timestamp" : "2019-06-10T09:53:34.636Z",
  "device" : {
    "screenSize" : "1200x1920",
    "appBuild" : "3",
    "osName" : "macOS",
    "timeZoneOffset" : 120,
    "osVersion" : "10.14.4",
    "locale" : "en_US",
    "appNamespace" : "co.boxyapp.contacts",
    "osBuild" : "18E226",
    "sdkName" : "appcenter.macos",
    "oemName" : "Apple",
    "model" : "MacBookPro14,3",
    "sdkVersion" : "2.0.1",
    "appVersion" : "1.0"
  },
  "type" : "startSession"
}
2019-06-10 11:53:37.648122+0200 Boxy for Contacts[21961:1556149] [AppCenterCrashes] VERBOSE: -[MSCrashes channel:didPrepareLog:internalId:flags:]/441 Storing a log to Crashes Buffer: (sid: B335CCFA-49BD-4B34-862F-16296B62ED27, type: event)
2019-06-10 11:53:37.648301+0200 Boxy for Contacts[21961:1556149] [AppCenterCrashes] VERBOSE: -[MSCrashes channel:didPrepareLog:internalId:flags:]/452 Found an empty buffer position.
2019-06-10 11:53:37.648663+0200 Boxy for Contacts[21961:1556377] [AppCenter] VERBOSE: -[MSAppCenterIngestion createRequest:eTag:authToken:]/91 URL: https://in.appcenter.ms/logs?api-version=1.0.0
2019-06-10 11:53:37.648761+0200 Boxy for Contacts[21961:1556377] [AppCenter] VERBOSE: -[MSAppCenterIngestion createRequest:eTag:authToken:]/92 Headers: Install-ID = 92F24F44-11CE-47C7-9F26-EA81F33AA2DC, App-Secret = ****************************6f7334e2, Content-Type = application/json
2019-06-10 11:53:37.649078+0200 Boxy for Contacts[21961:1556377] [AppCenter] DEBUG: -[MSChannelUnitDefault enqueueItem:flags:]_block_invoke/182 Saving log, type: event, flags: 1.
2019-06-10 11:53:37.654254+0200 Boxy for Contacts[21961:1556377] [AppCenter] VERBOSE: -[MSLogDBStorage saveLog:withGroupId:flags:]_block_invoke/120 Log is stored with id: '169'
2019-06-10 11:53:37.658276+0200 Boxy for Contacts[21961:1556377] [AppCenterCrashes] VERBOSE: -[MSCrashes channel:didCompleteEnqueueingLog:internalId:]/494 Deleting a log from buffer with id 5E3E72A4-E45F-467E-B09A-E6F5C564F9E2
2019-06-10 11:53:37.659301+0200 Boxy for Contacts[21961:1556377] [AppCenter] VERBOSE: -[MSLogDBStorage loadLogsWithGroupId:limit:excludedTargetKeys:afterDate:beforeDate:completionHandler:]/212 Load log(s) with id(s) '168' as batch Id:B0E0A0AA-8D85-471A-A348-5EE3B7E98DA3
2019-06-10 11:53:37.659471+0200 Boxy for Contacts[21961:1556377] [AppCenter] DEBUG: -[MSChannelUnitDefault sendLogContainer:withAuthTokenFromArray:atIndex:]/219 Sending 1/1 log, group Id: AppCenter, batch Id: B0E0A0AA-8D85-471A-A348-5EE3B7E98DA3, session Id: (null), payload:
{
  "device" : {
    "screenSize" : "1200x1920",
    "appBuild" : "3",
    "osName" : "macOS",
    "timeZoneOffset" : 120,
    "osVersion" : "10.14.4",
    "locale" : "en_US",
    "appNamespace" : "co.boxyapp.contacts",
    "osBuild" : "18E226",
    "sdkName" : "appcenter.macos",
    "oemName" : "Apple",
    "model" : "MacBookPro14,3",
    "sdkVersion" : "2.0.1",
    "appVersion" : "1.0"
  },
  "timestamp" : "2019-06-10T09:53:34.642Z",
  "type" : "startService",
  "services" : [
    "Crashes",
    "Analytics"
  ]
}
2019-06-10 11:53:37.659652+0200 Boxy for Contacts[21961:1556377] [AppCenter] VERBOSE: -[MSAppCenterIngestion createRequest:eTag:authToken:]/91 URL: https://in.appcenter.ms/logs?api-version=1.0.0
2019-06-10 11:53:37.659722+0200 Boxy for Contacts[21961:1556377] [AppCenter] VERBOSE: -[MSAppCenterIngestion createRequest:eTag:authToken:]/92 Headers: Install-ID = 92F24F44-11CE-47C7-9F26-EA81F33AA2DC, App-Secret = ****************************6f7334e2, Content-Type = application/json
2019-06-10 11:53:37.887622+0200 Boxy for Contacts[21961:1556370] [AppCenter] VERBOSE: -[MSHttpIngestion sendCallAsync:]_block_invoke/256 HTTP response received with status code: 200, payload:
CorrelationId: 8b77097d-f3c2-4b5f-80aa-198f238834e8  ReasonCode: Success
2019-06-10 11:53:37.887723+0200 Boxy for Contacts[21961:1556497] [AppCenter] DEBUG: -[MSChannelUnitDefault sendLogContainer:withAuthTokenFromArray:atIndex:]_block_invoke/242 Log(s) sent with success, batch Id:19157BE4-E70C-46A2-9EB2-166F9C1244F0.
2019-06-10 11:53:37.887801+0200 Boxy for Contacts[21961:1556370] [AppCenter] INFO: -[MSHttpIngestion call:completedWithResult:]/305 Removed call id:19157BE4-E70C-46A2-9EB2-166F9C1244F0 from pending calls:{
    "B0E0A0AA-8D85-471A-A348-5EE3B7E98DA3" = "<MSIngestionCall: 0x6000026212c0>";
}
2019-06-10 11:53:37.888030+0200 Boxy for Contacts[21961:1556377] [AppCenter] VERBOSE: -[MSHttpIngestion sendCallAsync:]_block_invoke/256 HTTP response received with status code: 200, payload:
CorrelationId: 6a8445b2-f94f-4752-91e2-8e309b8c4910  ReasonCode: Success
2019-06-10 11:53:37.888106+0200 Boxy for Contacts[21961:1556377] [AppCenter] INFO: -[MSHttpIngestion call:completedWithResult:]/305 Removed call id:B0E0A0AA-8D85-471A-A348-5EE3B7E98DA3 from pending calls:{
}
2019-06-10 11:53:37.895218+0200 Boxy for Contacts[21961:1556497] [AppCenter] VERBOSE: +[MSLogDBStorage deleteLogsFromDBWithColumnValues:columnName:inOpenedDatabase:]/353 Deletion of log(s) by id with value(s) '167' succeeded.
2019-06-10 11:53:37.895355+0200 Boxy for Contacts[21961:1556497] [AppCenter] DEBUG: -[MSChannelUnitDefault sendLogContainer:withAuthTokenFromArray:atIndex:]_block_invoke/242 Log(s) sent with success, batch Id:B0E0A0AA-8D85-471A-A348-5EE3B7E98DA3.
2019-06-10 11:53:37.896980+0200 Boxy for Contacts[21961:1556497] [AppCenter] VERBOSE: +[MSLogDBStorage deleteLogsFromDBWithColumnValues:columnName:inOpenedDatabase:]/353 Deletion of log(s) by id with value(s) '168' succeeded.
didFinish navigation
2019-06-10 11:53:40.922666+0200 Boxy for Contacts[21961:1556370] [AppCenter] VERBOSE: -[MSLogDBStorage loadLogsWithGroupId:limit:excludedTargetKeys:afterDate:beforeDate:completionHandler:]/212 Load log(s) with id(s) '169' as batch Id:76706226-684D-479D-BAB7-56797CD26CA8
2019-06-10 11:53:40.922971+0200 Boxy for Contacts[21961:1556370] [AppCenter] DEBUG: -[MSChannelUnitDefault sendLogContainer:withAuthTokenFromArray:atIndex:]/219 Sending 1/1 log, group Id: Analytics, batch Id: 76706226-684D-479D-BAB7-56797CD26CA8, session Id: B335CCFA-49BD-4B34-862F-16296B62ED27, payload:
{
  "device" : {
    "screenSize" : "1200x1920",
    "appBuild" : "3",
    "osName" : "macOS",
    "timeZoneOffset" : 120,
    "osVersion" : "10.14.4",
    "locale" : "en_US",
    "appNamespace" : "co.boxyapp.contacts",
    "osBuild" : "18E226",
    "sdkName" : "appcenter.macos",
    "oemName" : "Apple",
    "model" : "MacBookPro14,3",
    "sdkVersion" : "2.0.1",
    "appVersion" : "1.0"
  },
  "id" : "1A779874-FDFE-4AD7-9E37-7A37F0F44F68",
  "typedProperties" : [
    {
      "type" : "string",
      "name" : "theme",
      "value" : "automatic"
    },
    {
      "type" : "string",
      "name" : "statusItemEnabled",
      "value" : "true"
    },
    {
      "type" : "string",
      "name" : "numberOfAccounts",
      "value" : "5"
    }
  ],
  "sid" : "B335CCFA-49BD-4B34-862F-16296B62ED27",
  "timestamp" : "2019-06-10T09:53:37.647Z",
  "type" : "event",
  "name" : "AppLaunch"
}
2019-06-10 11:53:40.923881+0200 Boxy for Contacts[21961:1556370] [AppCenter] VERBOSE: -[MSAppCenterIngestion createRequest:eTag:authToken:]/91 URL: https://in.appcenter.ms/logs?api-version=1.0.0
2019-06-10 11:53:40.923958+0200 Boxy for Contacts[21961:1556370] [AppCenter] VERBOSE: -[MSAppCenterIngestion createRequest:eTag:authToken:]/92 Headers: Install-ID = 92F24F44-11CE-47C7-9F26-EA81F33AA2DC, App-Secret = ****************************6f7334e2, Content-Type = application/json
2019-06-10 11:53:41.062461+0200 Boxy for Contacts[21961:1556497] [AppCenter] VERBOSE: -[MSHttpIngestion sendCallAsync:]_block_invoke/256 HTTP response received with status code: 200, payload:
CorrelationId: 06fb70d4-315c-47d2-80ad-54da48dc1cd9  ReasonCode: Success
2019-06-10 11:53:41.062549+0200 Boxy for Contacts[21961:1556497] [AppCenter] INFO: -[MSHttpIngestion call:completedWithResult:]/305 Removed call id:76706226-684D-479D-BAB7-56797CD26CA8 from pending calls:{
}
2019-06-10 11:53:41.062584+0200 Boxy for Contacts[21961:1556377] [AppCenter] DEBUG: -[MSChannelUnitDefault sendLogContainer:withAuthTokenFromArray:atIndex:]_block_invoke/242 Log(s) sent with success, batch Id:76706226-684D-479D-BAB7-56797CD26CA8.
2019-06-10 11:53:41.070777+0200 Boxy for Contacts[21961:1556377] [AppCenter] VERBOSE: +[MSLogDBStorage deleteLogsFromDBWithColumnValues:columnName:inOpenedDatabase:]/353 Deletion of log(s) by id with value(s) '169' succeeded.
```

**Comment by MatkovIvan** [comment]
Hi @frankdilo!

Thanks for getting in touch!
It has already been fixed in #1607 and will be available in June release (2.1.0). I leave this issue open until release for visibility.

Best,
Ivan

**Comment by v-eacaoi** [comment]
Hey!
We've included fixes in the latest SDK release for this. I'm closing the issue but please reopen if you run into this again.
